### PR TITLE
Lvnodcdxi 18

### PR DESCRIPTION
**Pull request by StanislavFedyna**
created additional settings component

### CODE REVIEW

File: src/client/components/additionalConfigs/additionalConfigs.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faVolumeUp, faCog } from '@fortawesome/free-solid-svg-icons';
+
+import classes from './additionalConfigs.css';
+
+const volumeIcon = <FontAwesomeIcon icon={faVolumeUp} />;
+const cogIcon = <FontAwesomeIcon icon={faCog} />;
+
+class AdditionalConfigs extends Component {
```
Comment (tonypntsrn) [inline src/client/components/additionalConfigs/additionalConfigs.js]: If you are not using state of component no need to use class component here
Functional component will be enough

File: src/client/components/mainFooter/mainFooter.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './mainFooter.css';
+
+class MainFooter extends React.Component {
```
Comment (tonypntsrn) [inline src/client/components/mainFooter/mainFooter.js]: Same as in comment before, please also check your all components

Comment (StanislavFedyna) [reply src/client/components/mainFooter/mainFooter.js]: done)

**Review by tonypntsrn — APPROVED** [review]
👍

**Pull request merged**